tests/attestation_cert_apis/enc/enc.cpp

```diff
@@ -266,6 +316,14 @@ oe_result_t get_tls_cert_signed_with_key(
         "\nFrom inside enclave: verifying the certificate... %s\n",
         result == OE_OK ? "Success" : "Fail");
 
+    // validate cert with oe_verify_attestation_certificate_with_evidence() to
+    // ensure that the added report verifier part of the function works well
+    result = oe_verify_attestation_certificate_with_evidence(
+        output_cert, output_cert_size, sgx_enclave_claims_verifier, NULL);
+    OE_TRACE_INFO(
+        "\nFrom inside enclave: verifying the certificate... %s\n",
```

Comment:
Make this log different from line 316 so it's easier to tell which function fails.